pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -357,6 +374,16 @@ func GetPipelineConditionStatus(pr *v1alpha1.PipelineRun, state PipelineRunState
 
 	// A single failed task mean we fail the pipeline
 	for _, rprt := range state {
+		if rprt.IsCancelled() { //IsDone ensures we have crossed the retry limit
+			logger.Infof("TaskRun %s has cancelled, so PipelineRun %s has cancelled", rprt.TaskRunName, pr.Name)
```

Comment:
v minor nit: `has cancelled` -> `is cancelled`

Reply:
done